opte/src/engine/tcp.rs

```diff
@@ -278,7 +278,7 @@ impl TcpHdr {
             seq: 0,
             ack: 0,
             hdr_len_bytes: 20,
-            flags: 0x02,
+            flags: 0x00,
```

Comment:
Seems out of place, just checking you intended to keep this change.

Reply:
This was on purpose: I figured the flags should default to 0, not SYN (and it was causing some test failures IIRC).